src/tests/functional/plugin/shared/include/behavior/executable_network/exec_network_base.hpp

```diff
@@ -70,6 +70,10 @@ TEST_P(ExecutableNetworkBaseTest, checkGetMetric) {
 
 TEST_P(ExecutableNetworkBaseTest, canLoadCorrectNetworkToGetExecutableAndCheckConfig) {
     auto execNet = ie->LoadNetwork(cnnNet, target_device, configuration);
+    if (target_device == CommonTestUtils::DEVICE_AUTO) {
+        // AUTO executable network didn't support to read any config.
```

Comment:
probably EXPECT_THROW?

Reply:
Due to update of supported property list by `AutoExecutableModel `, which means we need to check if the property in the configuration passed from `LoadNetwork()` is supported by `AutoExecutableModel` and then decide to call `EXPECT_THROW `or `ASSERT_NO_THROW `to complete this test case. Actual the test case of `AutoExecutableModel` supported properties checking has been implemented in `src/tests/functional/plugin/shared/src/behavior/ov_executable_network/get_metric.cpp`